config_src/drivers/nuopc_cap/mom_cap.F90

```diff
@@ -890,7 +891,7 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
   call fld_list_add(fldsFrOcn_num, fldsFrOcn, "So_dhdy"    , "will provide")
   call fld_list_add(fldsFrOcn_num, fldsFrOcn, "Fioo_q"     , "will provide")
   call fld_list_add(fldsFrOcn_num, fldsFrOcn, "So_bldepth" , "will provide")
-  if (cesm_coupled) then
+  if (cesm_coupled .and. use_MARBL) then
```

Comment:
Same question as above, maybe we should remove the `cesm_coupled` part of this logical?